Pubsub:  broken by #6211

See https://source.cloud.google.com/results/invocations/99e4cfc1-98bf-4301-9191-27e6cac9412a/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fpubsub/log.

```python
__________________________________ test_open ___________________________________

thing = <module 'google.cloud.pubsub_v1.subscriber._protocol' from '/tmpfs/src/github/google-cloud-python/pubsub/google/cloud/pubsub_v1/subscriber/_protocol/__init__.py'>
comp = 'bidi', import_path = 'google.cloud.pubsub_v1.subscriber._protocol.bidi'

    def _dot_lookup(thing, comp, import_path):
        try:
>           return getattr(thing, comp)
E           AttributeError: module 'google.cloud.pubsub_v1.subscriber._protocol' has no attribute 'bidi'
.nox/default/lib/python3.6/site-packages/mock/mock.py:1197: AttributeError

During handling of the above exception, another exception occurred:

args = (), keywargs = {}
extra_args = [<MagicMock name='Heartbeater' spec='Heartbeater' id='140683705172264'>, <MagicMock name='Dispatcher' spec='Dispatcher' id='140683694890848'>, <MagicMock name='Leaser' spec='Leaser' id='140683694889448'>]
entered_patchers = [<mock.mock._patch object at 0x7ff37bea4198>, <mock.mock._patch object at 0x7ff37bea4be0>, <mock.mock._patch object at 0x7ff37bea4cc0>]
exc_info = (<class 'ModuleNotFoundError'>, ModuleNotFoundError("No module named 'google.cloud.pubsub_v1.subscriber._protocol.bidi'",), <traceback object at 0x7ff379ab3808>)
patching = <mock.mock._patch object at 0x7ff37bea4198>
arg = <MagicMock name='Leaser' spec='Leaser' id='140683694889448'>

    @wraps(func)
    def patched(*args, **keywargs):
        extra_args = []
        entered_patchers = []

        exc_info = tuple()
        try:
            for patching in patched.patchings:
>               arg = patching.__enter__()

.nox/default/lib/python3.6/site-packages/mock/mock.py:1297:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
.nox/default/lib/python3.6/site-packages/mock/mock.py:1353: in __enter__
    self.target = self.getter()
.nox/default/lib/python3.6/site-packages/mock/mock.py:1523: in <lambda>
    getter = lambda: _importer(target)
.nox/default/lib/python3.6/site-packages/mock/mock.py:1210: in _importer
    thing = _dot_lookup(thing, comp, import_path)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

thing = <module 'google.cloud.pubsub_v1.subscriber._protocol' from '/tmpfs/src/github/google-cloud-python/pubsub/google/cloud/pubsub_v1/subscriber/_protocol/__init__.py'>
comp = 'bidi', import_path = 'google.cloud.pubsub_v1.subscriber._protocol.bidi'

    def _dot_lookup(thing, comp, import_path):
        try:
            return getattr(thing, comp)
        except AttributeError:
>           __import__(import_path)
E           ModuleNotFoundError: No module named 'google.cloud.pubsub_v1.subscriber._protocol.bidi'

.nox/default/lib/python3.6/site-packages/mock/mock.py:1199: ModuleNotFoundError
```

--- a/pubsub/tests/unit/pubsub_v1/subscriber/test_streaming_pull_manager.py
+++ b/pubsub/tests/unit/pubsub_v1/subscriber/test_streaming_pull_manager.py
@@ -272,10 +272,10 @@ def test_heartbeat_inactive():
 
 
 @mock.patch(
-    'google.cloud.pubsub_v1.subscriber._protocol.bidi.ResumableBidiRpc',
+    'google.api_core.bidi.ResumableBidiRpc',
     autospec=True)
 @mock.patch(
-    'google.cloud.pubsub_v1.subscriber._protocol.bidi.BackgroundConsumer',
+    'google.api_core.bidi.BackgroundConsumer',
     autospec=True)
 @mock.patch(
     'google.cloud.pubsub_v1.subscriber._protocol.leaser.Leaser',

